hack/build.sh

```diff
@@ -3,7 +3,81 @@
 # This script builds the application from source for multiple platforms.
 set -e
 
+# Get the git commit
+getGitCommit()
+{
+    if [ -f $GOPATH/src/github.com/openebs/node-disk-manager/GITCOMMIT ];
+    then
+	GIT_COMMIT="`cat $GOPATH/src/github.com/openebs/node-disk-manager/GITCOMMIT`"
+    else
+	GIT_COMMIT="$(git rev-parse HEAD)"
+    fi
+    echo ${GIT_COMMIT}
+}
+
+# Delete the old contents
+deleteOldContents(){
+    rm -rf bin/*
+    mkdir -p bin/
+
+    if [ -z "${CTLNAME}" ];
+    then
```

Comment:
can we make separate function which setting CTLNAME and delete old contents of bin .

Reply:
Right @shovanmaity . My bad !!